features/ownership/approve.feature

```diff
@@ -19,6 +19,7 @@ Feature: Test approving ownership
     And the JSON response at "ownerEmail" should be "dev+e2etest@publiq.be"
     And the JSON response at "requesterId" should be "auth0|64089494e980aedd96740212"
     And the JSON response at "state" should be "approved"
+    And the JSON response at "approvedById" should be "7a583ed3-cbc1-481d-93b1-d80fff0174dd"
```

Comment:
Why is only the id checked and not the email?

Reply:
Because admins don't have an email in Keycloak.
If you read a little bit lower you will see the "Approving ownership of an organizer as creator" does check the email address.